website_supplier/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+    
+================
+Website Supplier
```

Comment:
I guess its better to use `Website Suppliers Page` as name, it describes better the feature.